pyttb/tensor.py

```diff
@@ -111,25 +137,42 @@ def from_function(
         shape: Tuple[int, ...],
     ) -> tensor:
         """
-        Creates a tensor from a function handle and size
+        Construct a :class:`pyttb.tensor` whose data entries are set using
+        a function.
 
         Parameters
         ----------
-        function_handle:
-            Function to generate data to construct tensor
+        fun: function, required
```

Comment:
The argument in tensor is `function_handle` same with `sptensor`. Unless you find this terribly verbose we should update ktensor to match. Also `required` is unnecessary in the description since there is no default provided.

Reply:
Hmm I see later on we used `fun` instead of the verbose `function_handle` in tensor. We should probably unify globally or file an issue to do so.